service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/UpdateGcsBucketStepTest.java

```diff
@@ -119,7 +121,9 @@ public void testDoStep() throws InterruptedException, RetryException {
 
     final LifecycleCondition rule2condition = rules.get(1).getCondition();
     assertEquals(15, rule2condition.getAge());
-    assertEquals(toGoogleDateTime(OFFSET_DATE_TIME_1), rule2condition.getCreatedBefore());
+    assertEquals(
```

Comment:
make a comment and reference PF-933 about the timestamps

Reply:
done